### DISCUSSION TITLE
Changelog for new releases?

### DISCUSSION THREAD

**Discussion by m-ueberall** [plantuml/plantuml#1140] [General]
Given that the last releases [v1.2022.9](https://github.com/plantuml/plantuml/releases/tag/v1.2022.9), [v1.2022.10](https://github.com/plantuml/plantuml/releases/tag/v1.2022.10) have not yet been listed in https://plantuml.com/news#external, is it possible to include a (compressed) changelog below the version for published releases? Thanks in advance!
**EDIT:** A short link to https://github.com/plantuml/plantuml/compare/v1.2022.9...v1.2022.10 would already be helpful.

**Comment by arnaudroques** [comment]
Sorry for the inconvenience but we have to do several "dummy" releases to test publishing to OSSRH ( see https://github.com/plantuml/plantuml/pull/1135#issuecomment-1260180863 )
Here is no real change since `v1.2022.8` so you can stick to it.
I think that we may need some more "dummy" releases to fix publishing to OSSRH. If it takes too long, we'll publish a new to explain people what's going on.
